### DISCUSSION TITLE
Eye patch option for glasses

### DISCUSSION THREAD

**Discussion by WilliamRADFunk** [dicebear/dicebear#134] [Ideas]
Wanted to suggest an additional option for the glasses, especially on the micah library. Any chance for an eye patch?

**Comment by FlorianKoerner** [comment]
A cool idea. Currently, we have only taken Micah Lanier's work. If you get in touch with him on Figma, he may implement it.

https://www.figma.com/community/file/829741575478342595

**Comment by WilliamRADFunk** [comment]
The Figma Avatar maker for Micah did wonderful work, but I fear he may not have as much bandwidth for adding new features.

I wonder if you had any plans for making a sort of modding to these child libraries you've wrapped into this library. Something where pull-requests would add to an addendum Figma that you draw from in future iterations of the library. Each child library like Micah would have it's own addendum area for additions in order to appropriately separate them, and being that they require pull requests for the library to recognize the new features, you'd still control when and how such new features rolled out.

(ie. adding a selection of hats to the micah avatars)

Maybe the Micah Figma doesn't allow for such a thing, I don't know. Just a thought.

**Reply by FlorianKoerner** [reply]
Sorry for the late reply. As soon as we provide a Figma version including an exporter plugin in addition to the avatar styles, modifications should be quite easy for everyone.

But until then it will probably take a little while. These modifications will probably not make it into the official packages. We want to keep our work as close as possible to the work of the designers.

But yes: In the future it should be possible for the community to provide modifications very easily. (Theoretically already possible now, just a bit more complex).
